app/views/admin/index.scala.html

```diff
@@ -322,7 +322,12 @@ <h1>Comments</h1>
                                     <td><a href='@routes.AdminController.userProfile(comment.username)'>@comment.username</a>
                                     </td>
                                     <td class = 'timestamp'>@comment.timestamp</td>
-                                    <td>@comment.gsvPanoramaId</td>
+                                    <td>
+                                        <a class="showCommentLocation" href="#" 
```

Comment:
Change from camelCase to kebab-case

Reply:
There are some other instances throughout this file of camelCase ids from older code, should I change those as well?